docker-storage-setup.sh

```diff
@@ -63,6 +63,10 @@ STORAGE_DRIVERS="devicemapper overlay"
 
 DOCKER_METADATA_DIR="/var/lib/docker"
 
+# DEVS can have device names without absolute path. Convert these to absolute
+# paths and save in ABS_DEVS and use in rest of the code.
+DEVS_ABS=""
+
```

Comment:
Being paranoid. Wanted to retain original names just in case we end up using them for output messages. Not very particular about it, but don't want to lose track of what user passed in originally.